app/views/posts/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>This is index page of posts</h1>
+<%= render 'shared/user', user: @user%>
+<ul>
+  <% if @user.posts.empty?%>
+    <p>No posts</p>
+  <%else%>
+  <%@user.posts.each do |post|%>
+    <li>
+      <%=render 'shared/post', post: post%>
+      <%= render 'shared/comment', comments: post.recent_comments%>
+    </li>
+  <%end%>
+  <%end%>
+</ul>
+<div class='see-all-post-btn-container'><a class='see-all-post-btn'>Pagination</a></div>
```

Comment:
- [ ] An error occurs while trying to show all posts that belong to a user. Kindly see the error below and try to fix it 👇 
![image](https://github.com/BranBayou/ror-blog-app/assets/46839548/0f945bf7-c0ac-401a-91b2-bf3343ba9ed4)